src/guides/v2.4/config-guide/cache/two-level-cache.md

```diff
@@ -21,28 +21,30 @@ Magento stores the hashed data version in Redis, with the suffix ':version' appe
 ```php
 'cache' => [
     [
```

Comment:
We may still have an error here. The cache node should not contain another array. Please, correct me if I'm wrong
```suggestion

```